### DISCUSSION TITLE
Failed when synchronizing using Edge.

### DISCUSSION THREAD

**Discussion by Liu8Can** [Tampermonkey/tampermonkey#2205] [Q&A]
When I synchronize Tampermonkey on Edge, it prompts "sycn failed". I have tried reinstalling the Tampermonkey plugin, but the problem still exists. How can I troubleshoot it?

**Comment by derjanb** [comment]
Please attach a screeshot. Thanks.

**Reply by Liu8Can** [reply]
我尝试

> Please attach a screeshot. Thanks.

Sorry. I hope you can open these two picture links. I still haven't found out the problem after spending a lot of time. So I later started to use the Tampermonkey BETA version, and the problem disappeared.

https://filestore.community.support.microsoft.com/api/images/655fc6e5-7b1c-423c-9b7c-f5f19efb1918?upload=true&fud_access=hC1SxZhn7m%2FZQJkOIiOVstu10yTQgXS4A%2FDBzZTg8nbaCgIogkrcDydMeI5Y4za2dOqDdWtsG2JNS3E35V60i9TiGHR7STMpJHheeXuDvO8nwjUlqCBHhJ0NDvuYN7OSH5bCdtkqYguYfQN%2FPpwPzkuvU9zZqY6WCra0WUd6aT3QfW5c41uC8v4XnztqwVVufDtea8IebV8Vv2o09%2BdgNxIOLFLgygk7MaEQ3V2PWK5XR7KZDxMgTqCQAfLhSKP7TY%2Bv9o7CVw3lbwmWFCoSEq87vb33HPkYpxJBvHRoh9b2X7CtiHgvstbtd04yE2LBGukeMZBW93tPkhkmifI65KVdJVoZUv5ChTx4rb5oAW0PoCCsj31myZQ2aMBmfyBC1Ax0UbD9Ewps9JdRkqEk6bj9cjxFkFrUo11WTOY8%2BKE%3D

https://filestore.community.support.microsoft.com/api/images/9d26f5c1-0786-44a8-8abc-e14098e59084?upload=true&fud_access=hC1SxZhn7m%2FZQJkOIiOVstu10yTQgXS4A%2FDBzZTg8nbaCgIogkrcDydMeI5Y4za2dOqDdWtsG2JNS3E35V60i9TiGHR7STMpJHheeXuDvO8nwjUlqCBHhJ0NDvuYN7OSH5bCdtkqYguYfQN%2FPpwPzkuvU9zZqY6WCra0WUd6aT3QfW5c41uC8v4XnztqwVVufDtea8IebV8Vv2o09%2BdgNxIOLFLgygk7MaEQ3V2PWK4nq5RjjKNnKVQRGBhhMg%2BtsehsBuAJyyWWj0YpYyEbyN%2F3eII53tBes4Nzhx1O0BPU4HNi8M6ZWKck0LAIuP8v4R5qbHnZBGixGVYx35MffBaRmFslj8%2Bmby3akJ05pOLW%2B%2BaaagUxg387pke9QTLRXUcnZ3C0nmO6%2FjpvnWP97cj%2FJbEwXFg2DbZtw%2Fni94A%3D
